server/server.js

```diff
@@ -109,7 +109,14 @@ app.get('/*', (req, res) => {
   res.sendFile(path.join(__dirname, '../client/public/index.html'))
 })
 
-
+//Chat service
```

Comment:
And let's leave this out for now since it will be covered by what I'm working on